lib/muffin_man/v20201201/catalog_items.rb

```diff
@@ -0,0 +1,38 @@
+module MuffinMan
+  module V20201201
+    class CatalogItems < SpApiClient
```

Comment:
No sandbox support yet @gavinritchie if you know where to find, I can add support here

Reply:
I've added sandbox support again, and it works!